image_server/cloudformation.yml

```
@@ -156,17 +156,13 @@ Resources:
                 - s3:ListBucket
                 - s3:GetBucketAcl
                 Resource:
-                - !Sub "arn:aws:s3:::${Bucket}"
-                - arn:aws:s3:::hyp3-examples
-                - arn:aws:s3:::hyp3-pdc-data
+                - !Sub "arn:aws:s3:::*"
```

`!Sub` isn't needed anymore since we're using the ${Bucket} variable anymore. Would still work, but I think it's worth tidying up:
```suggestion
                - arn:aws:s3:::*
```